src/routers/groups-routes.ts

```diff
@@ -0,0 +1,33 @@
+import {Router} from 'express';
+import * as groupValidation from '../validation/group-validation-schemas';
+import {createValidator} from 'express-joi-validation';
+import GroupsController from '../controllers/groups.controller';
+
+const router = Router();
+const validator = createValidator();
+
+router.get('/', GroupsController.getGroups);
+
+router.get('/:id', GroupsController.getGroup);
+
+router.post(
+  '/',
+  validator.body(groupValidation.groupBodySchema),
+  GroupsController.createGroup
+);
+
+router.put(
+  '/:id',
+  validator.body(groupValidation.groupBodySchema),
+  GroupsController.editGroup
+);
+
+router.delete('/:id', GroupsController.deleteGroup);
+
+router.put(
+  '/addusers/:id',
```

Comment:
```suggestion
  ':id/add-users',
```
You modify particular group which is identified as `/groups/:id`. And the action you want to perform is `add-users`. 
so full path is `/groups/:id/add-users`. Later, if you need another action it's going to be `/groups/:id/another-action`.

So, your path always has the same format
`/groups`
`/groups/:id`
`/groups/:id/action`